extras/AFC_spool.py

```diff
@@ -255,6 +255,10 @@ def cmd_SET_SPOOL_ID(self, gcmd):
 
             self.set_spoolID(cur_lane, SpoolID)
 
+            # If the lane is currently loaded to the toolhead, update the active spool in Spoolman
+            if cur_lane.name == self.afc.current
```

Comment:
Forgot the colon with first suggestion :-|
```suggestion
            if cur_lane.name == self.afc.current:
```